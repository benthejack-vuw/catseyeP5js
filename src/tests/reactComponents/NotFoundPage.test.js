import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../reactComponents/NotFoundPage';

test('should render header correctly', ()=>{
    const wrapper = shallow(<NotFoundPage/>);
    expect(wrapper).toMatchSnapshot();
})