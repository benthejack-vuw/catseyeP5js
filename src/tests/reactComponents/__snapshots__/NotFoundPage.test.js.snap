// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<div>
  <p>
    404! 
    <Link
      to="/"
    >
      Go Home
    </Link>
  </p>
</div>
`;
